components/trello/card-updates.js

```diff
@@ -0,0 +1,73 @@
+const trello = require("https://github.com/PipedreamHQ/pipedream/components/trello/trello.app.js");
+const _ = require("lodash");
```

Comment:
You can save on deploy time by just [requiring `lodash.get` directly](https://www.npmjs.com/package/lodash.get):

```
const get = require('lodash.get');

```